### ISSUE TITLE
Over saturation and poor compatibility with other mods (dungeons and structures)

### ISSUE THREAD

**Issue by harmalom** [Melonslise/Subterranean-Wilderness#8]
great mod, but you should look for a way to exclude them from forming in structures and dungeons, it causes a terrible delay saturation, for example with the Dungeon Crawl mod your mod saturates the passages, making them impassable, and in those where there are peat generators, it generates a terrible and unplayable lag, and so on with other dungeons and subway structures. 
If you manage to solve this mod it will be epic

Translated with www.DeepL.com/Translator (free version)

**Comment by Goldenlion5648** [comment]
Adding on to this, maybe a config option could be added to adjust how frequently the caves generate?

**Comment by Corvus20** [comment]
I was going to mention Dungeon Crawl's passages being clogged, too. More often than not caves are irritating to get through, let alone dungeons. Having a way to desaturate the caves a little bit would be nice.

**Comment by Iv5an** [comment]
Yeah because of the jungle or bamboo vegetation causing lag is there anyway it cam be fix
